pip/qsharp/__init__.py

```diff
@@ -56,5 +56,6 @@
     "PauliNoise",
     "DepolarizingNoise",
     "BitFlipNoise",
-    "PhaseFlipNoise"
+    "PhaseFlipNoise",
+    "GlobalCallable",
```

Comment:
This essentially declares the "public" API for our package. Exposing `GlobalCallable` feels unnecessary though? You should be able to access it just fine for internal use (via `qsharp._native`) without exposing it through here.

Reply:
That's a good point. In some version of the changes, I think I had wanted this to be public so that users could do `isinstance(variable, qsharp.GlobalCallable)` but since that gets wrapped up a Python generated callable it should never come up. This is definitely an internal type now and doesn't need to be exposed.